aries-site/src/data/structures/components.js

```diff
@@ -352,7 +354,29 @@ export const components = [
     },
     relatedContent: ['TextInput', 'Forms', 'Select'],
   },
-
+  {
+    name: 'Notifications',
```

Comment:
Wondering if "Notification" should be under components and "Notifications" and "Status Indicators" under templates? Notifications being why and where these patterns applied within an application and then would reference "Notification" component page for the various flavors.

This is a bit of wondering aloud for discussion purposes. No changes for this in this PR.